src/components/AddItem.jsx

```diff
@@ -71,17 +81,17 @@ function AddItem() {
         <fieldset>
           <legend>Choose how soon you will buy this again</legend>
 
-          {radioButtonOptions.map((radioBut, index) => (
+          {radioButtonOptions.map((radioBtn, index) => (
```

Comment:
(speaking as a dev and not mentor/PM, so no need to change) I'm big fan of full wording (radioButton instead of radioBtn) since consistency helps in rare cases tries to do search through code and such.